src/components/TodoList.js

```diff
@@ -39,13 +37,13 @@ function TodoList() {
     }
 
     const removeTodo = id => {
-        const removeArr = [...todos].filter(todo => todo.id !== id);
+        const removeArr = [todos].filter(todo => todo.id !== id);
 
         setTodos(removeArr);
     };
 
     const completeTodo = id => {
-        let updatedTodos = todos.map(todo => {
+        let updatedTodos = todos.each(todo => {
```

Comment:
There is no .each in JS. You could ask him the difference between forEach and map.
https://code.tutsplus.com/tutorials/javascript-map-vs-foreach-when-to-use-each-one--cms-38365